### PR TITLE
Added SMSAPI documentation (new integration)

### PR DESCRIPTION
**Pull request by zen**
**Description:**

Added support for new notification component using polish SMSAPI service

**Pull request in [home-asistant](https://github.com/home-assistant/home-assistant) (if applicable):** WIll follow once it's ready (kind of chicken and egg problem)

https://github.com/home-assistant/home-assistant/pull/25134

## Checklist:

- [/ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [/ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9848"></a>

### CODE REVIEW

File: source/_components/smsapi.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: "SMSAPI"
+description: "Instructions on how to add user notifications to Home Assistant."
+logo: smsapi.png
+ha_category:
+  - Notifications
+ha_release: 0.96
```
Comment (klaasnicolaas) [inline source/_components/smsapi.markdown]:
```suggestion
ha_release: 0.97
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/smsapi.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/smsapi.markdown]: Done

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**